build.gradle

```diff
@@ -46,6 +46,8 @@ dependencies {
 	compile('com.h2database:h2')
 	compile('net.sourceforge.nekohtml:nekohtml')
 	compile('org.projectlombok:lombok:1.16.6')
+	compile('org.flywaydb:flyway-core:4.0')
+	compile('mysql:mysql-connector-java:5.1.38')
```

Comment:
Yes, otherwise starting the app will fail:

```
2016-04-19T09:28:33.63+0200 [APP/0]      ERR Caused by: org.cloudfoundry.reconfiguration.org.springframework.cloud.CloudException: No suitable database driver found for githubcla-db service 
2016-04-19T09:28:33.63+0200 [APP/0]      ERR 	at org.cloudfoundry.reconfiguration.org.springframework.cloud.service.relational.DataSourceCreator.getDriverClassName(DataSourceCreator.java:81)
2016-04-19T09:28:33.63+0200 [APP/0]      ERR 	at org.cloudfoundry.reconfiguration.org.springframework.cloud.service.relational.DataSourceCreator.create(DataSourceCreator.java:50)
2016-04-19T09:28:33.63+0200 [APP/0]      ERR 	... 24 more
```